studies/templates/studies/_study_fields.html

```diff
@@ -85,4 +186,12 @@
 {% bootstrap_field form.duration %}
 {% bootstrap_field form.contact_info %}
 {% bootstrap_field form.lab %}
-
+<div id="structure-container">
```

Comment:
Looking at the [django-bootstrap3 docs](https://django-bootstrap3.readthedocs.io/en/latest/templatetags.html#bootstrap3.templatetags.bootstrap3.bootstrap_field), shouldn't the field partial already render with error elements? If you really want to change the default class, I guess you could (see `error_css_class`), but I don't really see the point here

Reply:
Ooh, okay, I think my thinking was I shouldn't mess with the proper bootstrap error elements when adding client-side validation, to avoid interfering with errors added when actually submitting the form. Changing now.

Reply:
(The reason they're in their own divs is just to make the JS for identifying the appropriate editor elements more natural and less dependent on the details of how the ace-editor gets inserted/what IDs are used.)